test/functional/Java9andUp/modules/README.txt

```diff
@@ -0,0 +1,25 @@
+/*******************************************************************************
```

Comment:
Could convert to .md format and use this may be a good opportunity to add info about what the actual test case(s) do, with an accompanying diagram.  Essentially a brief statement on how modules should work, and how the test works.  

That way, if some new person comes in to augment testing or understand why their change broke this existing set of tests, they have a better understanding.